### ISSUE TITLE
Segmentation fault on Ubuntu

### ISSUE THREAD

**Issue by bmaz** [CERES-Sorbonne/Panoptic#5]
Hello @Tyrannas and @godicked 

I tried to start panoptic on Ubuntu 22 with python 3.10.8 and I got the following error when trying to select the image folder:

```bash
bmaz@ptl11206:~$ panoptic
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
INFO:     Started server process [20242]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     127.0.0.1:44350 - "GET /folders HTTP/1.1" 200 OK
Segmentation fault (core dumped)

```
Thanks for your help!

**Comment by Tyrannas** [comment]
This was indeed caused by pyQT setEnabled method which is crashing for unknown reason on linux. This should be fixed on `0.2.0`

**Comment by Tyrannas** [comment]
Indeed fixed on ubuntu
